### DISCUSSION TITLE
[Feature] Toggle panorama/360 view on and off

### DISCUSSION THREAD

**Discussion by Simonh2o** [immich-app/immich#3854] [Feature Request]
### The feature

Small feature request: Ability to toggle 360 view of panoramas on and off with an account wide setting, or simply a small button on the current image. Sometimes the panorama just looks better when displayed in full.

### Platform

- [X] Web
- [ ] Server
- [ ] Mobile

**Comment by bvonbla** [comment]
Yes please.
There seems to  be a problem with some systems where webgl is not working correctly and panormas are displayed as 'a black screen' That issue is not an issue with immich but with the drivers on those machines.
So a toggle to switch it of would be nice!

I was able to 'switch it off'  bij updating the exif-data on all panorama images in my library. Immich detects panorma images by checking if the exif field "ProjectionType" equals "equirectangular" So i had a script change that field to "panorama"  on all my panoramas. If it gets fixed i can have the script change it back to "equirectangular"
https://github.com/immich-app/immich/blob/702e91145ad51cdea813cecf879982ebac709141/web/src/lib/components/assets/thumbnail/thumbnail.svelte#L152

**Comment by cipriantudoran** [comment]
Any news, will this be implemented? It's really needed.
